lib/tasks/admin.rake

```diff
@@ -0,0 +1,15 @@
+namespace :admin do
```

Comment:
Wouldn't it be better to have `rake autolab:promote_user` since we already have that name space for other autolab related tasks?